scripts/asr_language_modeling/ngram_lm/train_kenlm.py

```diff
@@ -81,6 +81,21 @@ def main():
     parser.add_argument(
         "--do_lowercase", action='store_true', help="Whether to apply lower case conversion on the training text"
     )
+    parser.add_argument(
+        "-T",
+        default=None,
+        help="Provide non-default temporary directory for KenLM's lmplz. Useful if /tmp runs out of space during training.",
+    )
+    parser.add_argument(
+        "--limit_vocab_file",
+        default=None,
+        help="see lmplz --limit_vocab_file documentation. https://github.com/kpu/kenlm/blob/bcd4af619a2fa45f5876d8855f7876cc09f663af/lm/builder/lmplz_main.cc#L107",
+    )
+    parser.add_argument(
```

Comment:
As NeMo currently does not support word-level KenLM, what is the purpose of training such a model with nemo?
The Riva users may use the training scripts provided by the Riva right?

Reply:
The short answer is that there are some ASR users that aren't using Riva.

With more elaboration, CTC model decoding is fastest when using a WFST decoder on the GPU and people want this outside of Riva. In addition, whenever non-CTC models come (in particular, from K2 experiments), it is smart to use a WFST decoder because it is generic. There will be no need to manually create a decoder for each topology you happen to train with.

Of course, I'm not super happy about just adding on options to existing code in general; this was just a first draft to unblock someone.

Reply:
Hmm it seems like it will invite confusion. Also word LMs occupy way too much storage space + training time. We should have a seperate arg to make it a users explicit choice to train such model. Defaulting to char or subword LM otherwise.